lib/renderer/extensions/i18n.js

```diff
@@ -6,9 +6,10 @@
 // Does not implement predefined messages:
 // https://developer.chrome.com/extensions/i18n#overview-predefined
 
+const { nativeRequire } = require('@electron/internal/renderer/remote')
```

Comment:
why is this called `nativeRequire`...? it isn't native require.

Reply:
@miniak I agree, it’s a bad name. Any suggestions? I can rename it in another PR.

Reply:
Perhaps `potentiallyRemoteRequire`? `requirePrivileged`? `unsandboxedRequire`?

Reply:
`potentiallyRemoteRequire` it is, sounds scary as it should, to deter people from using it

Reply:
https://github.com/electron/electron/pull/15971